rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -56,7 +56,7 @@
 %endif
 
 # 2 - RHEL & Fedora
-%if 0%{?rhel} && 0%{?rhel} <= 6
+%if 0%{?rhel} && 0%{?rhel} <= 8
```

Comment:
we want to keep version 6 as limit for pie

Reply:
yes indeed dumb dev

Reply:
fixed